### DISCUSSION TITLE
IsEqual IsNotEqual Samples

### DISCUSSION THREAD

**Discussion by StefanKoell** [AvaloniaUI/Avalonia#16419] [Q&A] [answered]
I've seen the new built-in converters (IsEqual and IsNotEqual) and I'm wondering if they can be used directly in XAML. If so, a sample on how to use it would be very helpful.

**Comment by timunie** [comment]
Maybe with `{Binding Converter={x:Static ObjectConvertes.IsEqual}}` but as I have no PC at hand rn can't look up exact syntax atm.

**Reply by StefanKoell** [reply]
Sorry I haven't been clearer. I know how to use a converter in general but the "special case" here is that this converter needs a comparison value in order to work. I tried to compare it to another instance of my viewmodel like `ConverterParameter={Binding otherProp}` but this doesn't seem to work. A couple of samples on how to make use of this new converters would be very helpful.

**Reply by timunie** [reply, marked as answer]
You can't bind ConverterParameter. Either MultiBinding or a static instance to compare to is required

**Reply by StefanKoell** [reply]
That's very useful information, thank you! Didn't know this limitation and also couldn't find any info in the docs.

**Reply by timunie** [reply]
feel free to add a note to the docs
